### PR TITLE
Denis/yoext 94

### PR DESCRIPTION
**Pull request by Nebyt**
Moved the mock dApp page methods to the separate file.

### CODE REVIEW

File: packages/yoroi-extension/features/connector-cardano.feature [hunk 1]
```diff
@@ -10,7 +10,7 @@ Feature: dApp connector data signing
     Then I should see the connector popup
     And I select the only wallet named shelley-simple-15 with 5.5 balance
     Then The popup window should be closed
-    And The access request should succeed
+    And The access request should be succeed
```
Comment (yushih) [inline packages/yoroi-extension/features/connector-cardano.feature]: Just nitpicking: "succeed" is a verb so it should be "succeed" or "be successful".

**Review by yushih — APPROVED** [review]

File: packages/yoroi-extension/features/connector-cardano.feature (hunk 1 above)
Comment (Nebyt) [reply packages/yoroi-extension/features/connector-cardano.feature]: Got it. I will change it back.

**Review by Nebyt — COMMENTED** [review]
/check

**Pull request merged**